routes/database.rb

```diff
@@ -35,6 +35,7 @@ def self.registered(app)
           MyLog.log.fatal 'routes/database: Database Resource not found [edit]'
           halt 404, 'database resource not found'
         end
+        @project = Project.filter(id: @db.project_id).all.first
         @projects = Project.all
```

Comment:
i agree
It should be `@projects = Project.filter(id: @db.project_id).all` (without the first)
then in the view, it should iterate through the projects